docs/joss_paper/paper.md

```diff
@@ -55,67 +67,67 @@ bibliography: paper.bib
 
-![Registration network with unsupervised loss.\label{fig:unsupervised}](https://github.com/DeepRegNet/DeepReg/blob/master/docs/source/_images/registration-ddf-nn-unsupervised.svg?raw=true)
+![Registration network with unsupervised loss.\label{fig:unsupervised}](../source/_images/registration-ddf-nn-unsupervised.png)
 
 ## Weakly-supervised learning
-Weak supervision which utilises segmented corresponding regions in the medical image pairs was first proposed in a multimodal application for registering prostate MR to ultrasound images [@hu2018label]. In addition to the regularisation on the predicted displacement field, the training is driven by minimising the dissimilarity between the fixed labels and the warped moving labels. This formulation is modality-independent and the label dissimilarity measures are adapted from well-studied computer vision and medical imaging tasks, such as image segmentation. When multiple corresponding label pairs are available, a two-stage sampling strategy can be adopted for an unbiased gradient back-propagation during training [@hu2018weakly]. \autoref{fig:weakly} shows a schematic illustration of the weakly-supervised network training. Label dissimilarity measures include Dice, Jaccard, cross-entropy and their variants. `DeepReg` uses as a default multiscale Dice [@hu2018weakly], whilst other dissimilarity measures are implemented for users to tailor training options for various applications.
+Weak supervision which utilises segmented corresponding regions in the medical image pairs was first proposed in a multimodal application for registering prostate MR to ultrasound images [@hu2018label]. In addition to the regularisation on the predicted displacement field, the training is driven by minimising the dissimilarity between the fixed labels and the warped moving labels. This formulation is modality-independent and the label dissimilarity measures are adapted from well-studied computer vision and medical imaging tasks, such as image segmentation. When multiple corresponding label pairs are available, a two-stage sampling strategy can be adopted for an unbiased gradient back-propagation during training [@hu2018weakly]. \autoref{fig:weakly} shows a schematic illustration of the weakly-supervised network training. Label dissimilarity measures including Dice, Jaccard, square-mean-difference, cross-entropy and their variants. DeepReg uses as a default multiscale Dice [@hu2018weakly], whilst other dissimilarity measures are implemented for users to tailor training options for various applications.
```

Comment:
"square-mean-difference" or "sum-of-squared difference" ?

Reply:
i can't remember which one is in the code now.... :D